### DISCUSSION TITLE
VS Code reorders properties in ipynb file

### DISCUSSION THREAD

**Discussion by VacuousTruth0** [microsoft/vscode-jupyter#9601] [Questions and Answers] [answered]
1. Create an ipynb file with a code cell:
```
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "print('Hello world')"
   ]
  }
 ],
 "metadata": {
  "language_info": {
   "name": "python"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
```
2. Open the file in VS Code and save it.
3. The order of properties in the code cell has changed - `metadata` and `outputs` are now below `source`:
```
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "source": [
    "print('Hello world')"
   ],
   "outputs": [],
   "metadata": {
    "tags": []
   }
  }
 ],
 "metadata": {
  "language_info": {
   "name": "python"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
```

Is this the expected behaviour? Based on https://github.com/microsoft/vscode-jupyter/issues/742, I was expecting the properties to remain in sorted order.

I'm using VS Code 1.60.0.

**Comment by digitarald** [comment, marked as answer]
@VacuousTruth0 agreed, based on https://github.com/microsoft/vscode-jupyter/issues/742 your reordering scenario is unexpected. Moved to https://github.com/microsoft/vscode-jupyter/issues/9649.
